### ISSUE TITLE
Create notification template for Board of Appeals Higher Level Review Form Submission Confirmation

### ISSUE THREAD

**Issue by miabecker** [department-of-veterans-affairs/notification-api#523]
# Value Statement
**As** a Veteran or non-Veteran beneficiary
**I want to** receive a confirmation email when I submit a Decision Review Request: Board Appeal (VA Form 10182)
**So that**I know my form as been received 

# Acceptance Criteria
**GIVEN**  VA Form 10182
**WHEN** the form is submitted
**THEN** a confirmation email is sent to the end user

# Checklist
- [ ] Create in staging and post template ID to intake ticket linked below
- [ ] Create in production post template ID to intake ticket linked below

# Assumptions
-

# Additional Info/Resources

- Intake Ticket: https://app.zenhub.com/workspaces/vanotify-business-intake-board-606cc5c49392c900162c3971/issues/department-of-veterans-affairs/va.gov-team/26830

- Template Name: : Higher-Level Review VA Form 20-0996 Email Confirmation
- Service: VA.gov

- Content:

Dear Jane,

Thank you for filing a Higher-Level Review.

To check your Decision Review status, go to https://www.va.gov/claim-or-appeal-status/ to sign in to our claim status tool. Please allow 24 hours for your Higher-Level Review to show up there

Your Decision Review details are below:
Higher-Level Review

Date Submitted
June 29, 2021

If we have questions or need more information, we may contact you. We’ll notify you by mail once we have processed your Decision Review.

Thank you for your service,
VA

You are receiving this email because you filed a Higher-Level Review with VA. Please do not reply to this email. If you need to contact us please visit https://va.gov or by phone: 1-800-827-1000.

# Out of Scope
-

# Open Questions

**Comment by philherbert** [comment]
I've created the template in the `va.gov` service:

Staging: `1f747c23-1884-4424-9be7-161bc2e6f7e0`

**Comment by philherbert** [comment]
moving into blocked while we wait for feedback from the business line!

**Comment by miabecker** [comment]
Waiting on the HRL team to figure out where they want to trigger from light house or vets-api.

**Comment by philherbert** [comment]
whoops, template ID for staging under the correct service (lighthouse, as specified in the intake ticket):

`13dff506-1f54-4236-a123-f84622fdeb72`

**Comment by miabecker** [comment]
@anitadewitt-va this is ready to be created in production under lighthouse service!

**Comment by anitadewitt-va** [comment]
template ID for prod: `067d8f49-13f4-4f67-be6d-c1070881c952`
